### ISSUE TITLE
order keys should not be significant to deep equals.

### ISSUE THREAD

**Issue by dominictarr** [tapjs/tapjs#19]
this test should pass:

``` js
test('order of keys should not be significant', function (t) {
  t.deepEqual({a: 0, b: 0}, {b: 0, a: 0})
  t.end()
})
```

instead, output is: 

```
    found:  
      a: 0
      b: 0
    wanted: 
      b: 0
      a: 0
    diff:   |
      FOUND:  {"a":0,"b":0}
      WANTED: {"b":0,"a":0}
                ^ (at position = 2)
  ...

```

stringify(a) == stringify(b) is not generally they right way to test if two objects have the same structure.

**Comment by isaacs** [comment]
Sure.  Wanna write a patch?

**Comment by dominictarr** [comment]
okay, sure.

**Comment by ghost** [comment]
fixed in 0.1.4
